### ISSUE TITLE
LUA Error - 11.0.2.56162

### ISSUE THREAD

**Issue by stevedenunzio** [Stanzilla/PetJournal-Enhanced#11]
Don't think I did anything to trigger the error?

Message: Interface/AddOns/PetJournalEnhanced/PetJournalEnhanced.toc:37 Duplicate File Load Detected. [Interface/AddOns/PetJournalEnhanced/local-zhCN.lua] first loaded at [Interface/AddOns/PetJournalEnhanced/PetJournalEnhanced.toc:34]
Time: Mon Aug 19 12:34:45 2024
Count: 2

**Comment by yoshimo** [comment]
No, it is just a duplicate line in the toc file that triggers a warning these days.
If you remove one of the two copies of ```local-zhCN.lua``` in the toc file, the message will disappear
